EIPS/eip-6353.md

```diff
@@ -0,0 +1,333 @@
+    *@notice Remove the address from the whitelist and set rate to the default rate.
+    * @dev Requirements:
+     *
+     * - `toRemove` cannot be the zero address.
+     *
+     * @param toRemove The address to remove from whitelist.
+     */
+    function deleteFromWhitelist(address toRemove) external;
+
+    /**
+    *@notice Get all registered charity addresses.
+     */
+    function getAllWhitelistedAddresses() external ;
+
+    /**
+    *@notice Display for a user the rate of the default charity address that will receive donation.
+     */
+    function getRate() external view returns (uint256);
+
+    /**
+    *@notice Set personlised rate for charity address in {whitelistedRate}.
+    * @dev Requirements:
+     *
+     * - `whitelistedAddr` cannot be the zero address.
+     * - `rate` cannot be inferior to the default rate.
+     *
+     * @param whitelistedAddr The address to set as default.
+     * @param rate The personalised rate for donation.
+     */
+    function setSpecificRate(address whitelistedAddr , uint256 rate) external;
+
+    /**
+    *@notice Set for a user a default charity address that will receive donation. 
+    * The default rate specified in {whitelistedRate} will be applied.
+    * @dev Requirements:
+     *
+     * - `whitelistedAddr` cannot be the zero address.
+     *
+     * @param whitelistedAddr The address to set as default.
+     */
+    function setSpecificDefaultAddress(address whitelistedAddr) external;
+
+    /**
+    *@notice Set for a user a default charity address that will receive donation. 
+    * The rate is specified by the user.
+    * @dev Requirements:
+     *
+     * - `whitelistedAddr` cannot be the zero address.
+     * - `rate` cannot be inferior to the default rate 
+     * or to the rate specified by the owner of this contract in {whitelistedRate}.
+     *
+     * @param whitelistedAddr The address to set as default.
+     * @param rate The personalised rate for donation.
+     */
+    function setSpecificDefaultAddressAndRate(address whitelistedAddr , uint256 rate) external;
+
+    /**
+    *@notice Display for a user the default charity address that will receive donation. 
+    * The default rate specified in {whitelistedRate} will be applied.
+     */
+    function specificDefaultAddress() external view returns (
+        address defaultAddress
+    );
+
+    /**
+    *@notice Delete The Default Address and so deactivate donnations .
+     */
+    function deleteDefaultAddress() external;
+}
+
+interface IERC165 {
+    /// @notice Query if a contract implements an interface
+    /// @param interfaceID The interface identifier, as specified in ERC-165
+    /// @dev Interface identification is specified in ERC-165. This function
+    ///  uses less than 30,000 gas.
+    /// @return `true` if the contract implements `interfaceID` and
+    ///  `interfaceID` is not 0xffffffff, `false` otherwise
+    function supportsInterface(bytes4 interfaceID) external view returns (bool);
+}
+
```

Comment:
You can remove this. Interested readers can find EIP-165 from your `requires:` preamble field.

```suggestion
```